tracing-fmt/src/lib.rs

```diff
@@ -16,36 +16,48 @@ use tracing_core::{field, subscriber::Interest, Event, Metadata};
 
 use std::{any::TypeId, cell::RefCell, fmt, io};
 
+#[cfg(test)]
+#[macro_use]
+extern crate lazy_static;
```

Comment:
Very minor nit: I'd prefer just to import the macro in the test, rather than using `extern crate`, as I think it's not considered idiomatic in Rust 2018.

So, instead, you could just put:
```rust
use lazy_static::lazy_static;
```
in `writer::tests`.

Reply:
I was wondering how to do that without using `extern crate` but got confused when `use lazy_static` didn't work (didn't realize I had to do `use lazy_static::lazy_static`).